docker/Dockerfile

```diff
@@ -4,10 +4,11 @@ FROM openjdk:11.0-jre-slim
 # working directory when the docker is run
 WORKDIR /usr
 
-# install curl (to download install packages) and python (for gcloud sdk)
+# install curl (to download install packages), python (for gcloud sdk) and git
 RUN apt-get update \
     && apt-get install -y curl \
```

Comment:
can these all be on one line like `apt-get install curl python git`?

Reply:
good point